### ISSUE TITLE
Group Project: Social Mapping

### ISSUE THREAD

**Issue by cmccraw** [cmccraw/LiliElbe_EngagedLearners#14]
@doverebmann @m-nesbitt @joshbrockland @ProfPLC 

Use this Issue to communicate with Caroline and/or each other about your social mapping work outside of class.

**Comment by cmccraw** [comment]
@doverebmann @m-nesbitt @joshbrockland 

Here are some links that might be interesting as you start thinking about social mapping-- feel free to ask questions or add links etc. to ideas you want to bounce off of me!

https://rawgraphs.io/
https://cdh.unc.edu/resources/tools/
https://www.census.gov/dataviz/visualizations/stem/stem-html/
https://mina-loy.com/maps/loys-social-network/
https://www.tableau.com/

**Comment by cmccraw** [comment]
@doverebmann @m-nesbitt @joshbrockland

@RJP43 also recommended these two resources:
https://cytoscape.org/
https://www.kumu.io/ 

& these projects:
http://www.sixdegreesoffrancisbacon.com/?ids=10000473&min_confidence=60&type=network
http://nelson.newtfire.org/analysis.html (some of Becca's fab work!)

Becca can also make the raw text or the XML files for the _MIW_ edition available for you, so you could quickly search all the tagged places/people, their frequency, when/where they occur, etc.

**Comment by RJP43** [comment]
:wave:  
  
[XML to Cytoscape Tutorial](http://ebeshero.github.io/thalaba/cytosc.html) by @ebeshero

**Comment by m-nesbitt** [comment]
@RJP43 @cmccraw I am very interested in the custom templete on Kumu as a possibility for our social mapping. Do we know if we are able to put this type of project onto the website without making it a screenshot or URL link?

**Comment by cmccraw** [comment]
@m-nesbitt Yep it looks like they make it pretty easy to embed the map on a website— https://docs.kumu.io/guides/share-and-embed.html

Also, they do seem to have student discounts (mostly free?) if there are any paid features that jump out to y'all...

**Comment by cmccraw** [comment]
@m-nesbitt in general, their help docs seem pretty useful: https://docs.kumu.io/

**Comment by cmccraw** [comment]
@doverebmann @m-nesbitt @joshbrockland tons of resources listed here: https://guides.nyu.edu/dighum/tools

**Comment by m-nesbitt** [comment]
@cmccraw that's so exciting, thank you so much!

**Comment by cmccraw** [comment]
@doverebmann @m-nesbitt @JBrockland 

Am checking in with each group on GitHub, since so many folks were missing today; but y'all seemed to be doing great! 

Let me know if there's anything I can do to help, or if having the raw text or XML for the editions would be helpful. 

I would also say to begin thinking about some contextual text that might go with your map on the website (which maybe you already have!). Just some explanatory language describing how a reader/user might navigate the information that you're presenting, etc.
